app/main/views/admin_manager.py

```diff
@@ -28,3 +31,33 @@ def manage_admin_users():
 
     return render_template("view_admin_users.html",
                            admin_users=admin_users)
+
+
+@main.route('/admin-users/invite', methods=['GET', 'POST'])
+@role_required('admin-manager')
+def invite_admin_user():
+    form = InviteAdminForm(request.form)
+
+    if request.method == 'POST' and form.validate_on_submit():
+        email_address = form.data.get('email_address')
+        role = form.data.get('role')
+
+        send_user_account_email(
+            role,
+            email_address,
+            '08ab7791-6038-4ad2-9560-740bbcb675b7',
+            personalisation={'name': current_user.name}
+        )
+        flash('An invitation has been sent to {}.'.format(email_address), category='success')
+        return redirect(url_for('main.manage_admin_users'))
+
+    errors = {
+        key: {'question': form[key].label.text, 'input_name': key, 'message': form[key].errors[0]}
+        for key, value in form.errors.items()
+    }
```

Comment:
I noticed on the form error screengrab that the errors in the header are listed out of order. I only mention this as I just fixed the equivalent error on the content loader! https://github.com/alphagov/digitalmarketplace-content-loader/pull/52/ 

Completely up to you whether you think it's worth bothering to sort a very basic list of two errors 😸